src/components/NavBar/navBar.tsx

```diff
@@ -27,30 +41,18 @@ function NavBar() {
         </Link>
         <Navbar.Toggle aria-controls="basic-navbar-nav" />
         <Navbar.Collapse id="basic-navbar-nav" className="justify-content-end">
-          <Nav className="navbar navbar-light" >
+          <Nav className="navbar" >
             <Link to="/about-us" className={linkDefault} activeClassName={linkActive} >About Us</Link>
             <NavDropdown title="Sponsors">
-              <NavDropdown.Item>
-                <Link to="/sponsors" className={dropDefault} activeClassName={linkActive} > Current Sponsors</Link>
-              </NavDropdown.Item>
-              <NavDropdown.Item>
-                <Link to="/how-to-be-sponsors" className={dropDefault} activeClassName={linkActive} > Become a Sponsor</Link>
-              </NavDropdown.Item>
+              <DropdownItem {...DropdownItemProps} to="/sponsors">Current Sponsors</DropdownItem>
```

Comment:
This now produces a result like `<NavDropdown.Item as={Link} ... >`. Now, the entire item is clickable rather than just the text.